packages/webapp/src/elections/api/eden-contract.ts

```diff
@@ -145,6 +152,51 @@ const getVoteDataRows = async (
 
 export const getVoteData = getVoteDataRows;
 
+const getCommonDelegateAccountForGroupWithThisMember = (
+    round: number,
+    member?: EdenMember
```

Comment:
Looks like everything calling this function _has_ the `member`. Functions calling this function have the member as a required param. Does this need to be optional here?

Reply:
good call; question mark removed